### DISCUSSION TITLE
Read data display on screen to save SD

### DISCUSSION THREAD

**Discussion by phamhuyvn** [Bodmer/TFT_eSPI#2919] [Q&A - General]
I have a project that I need to know what display on my TFT, 
Anyway for me to get data which is display on screen and save to file or array to SD.

Thank

**Comment by Bodmer** [comment]
Not all displays support reading back the graphics RAM.

Run the example here to see if readPixel works:
https://github.com/Bodmer/TFT_eSPI/tree/master/examples/Test%20and%20diagnostics/TFT_ReadWrite_Test

If that does not work then drawing the screen into a sprite is an option if sufficient RAM is available.

**Reply by phamhuyvn** [reply]
THanks, I can get pixel but it too low. Anyway we can do anything in a buffer / frame and then display it to screen then
